TROUBLESHOOTING.md

```diff
@@ -8,18 +8,18 @@ with the same Ruby version as your projects, we need to properly set these envir
 invoking your Ruby version manager.
 
 The extension runs a command using your shell's interactive mode, so that version managers configured in files such as
-`~/.zshrc` are picked up automatically. The command then exports the environment information into JSON, so that we can
+`~/.zshenv` are picked up automatically. The command then exports the environment information into JSON, so that we can
```

Comment:
I think it's dependent of your ZSH setup. I run ZSH with [Prezto](https://github.com/sorin-ionescu/prezto). My ruby is sourced in `~/.zshrc` with Mise and it works. So i'm not sure the doc change cover all cases.

Reply:
@vinistock I created this file when testing this functionality. 

@louim I attempted to source from `~/.zshrc` and it failed to initialize. :/

Reply:
per [zsh docs](https://zsh.sourceforge.io/Guide/zshguide02.html#l9) it will run `/etc/zprofile` after `~/.zshenv` which, on recent versions of  macOS at least, does things like put the system ruby ahead of rbenv, asdf etc in PATH.

I had to do asdf init, for example, inside my `~/.zprofile` which gets sourced _after_ `/etc/zprofile`. I'm not particularly knowledgable here, but the docs suggest this depends on how zsh is invoked and for what type of session.